staging/src/k8s.io/code-generator/cmd/client-gen/generators/client_generator.go

```diff
@@ -130,7 +130,7 @@ func DefaultNameSystem() string {
 }
 
 func packageForGroup(gv clientgentypes.GroupVersion, typeList []*types.Type, clientsetPackage string, groupPackageName string, groupGoName string, apiPath string, srcTreePath string, inputPackage string, boilerplate []byte) generator.Package {
-	groupVersionClientPackage := strings.ToLower(filepath.Join(clientsetPackage, "typed", groupPackageName, gv.Version.NonEmpty()))
+	groupVersionClientPackage := filepath.Join(clientsetPackage, "typed", groupPackageName, strings.ToLower(gv.Version.NonEmpty()))
```

Comment:
Can you lower case the group package name as well?  Here and other places.

I don't see any reason why mixed case is needed there.

Reply:
addressed in the latest updates.